cfme/tests/cloud/test_providers.py

```diff
@@ -82,7 +82,7 @@ def test_provider_crud(provider):
         test_flag: crud
     """
     provider.create()
-    provider.validate(db=False)
+    provider.validate(ui=True)
```

Comment:
Well... yeh, if the validate itself was variable.
But the way its written now, where the `num_*` methods are variable, it should be like this.